diff --git a/pkgs/development/tools/gnulib/default.nix b/pkgs/development/tools/gnulib/default.nix
index 6c0883d..6db8dcb 100644
--- a/pkgs/development/tools/gnulib/default.nix
+++ b/pkgs/development/tools/gnulib/default.nix
@@ -8,7 +8,8 @@ stdenv.mkDerivation {
   src = fetchgit {
     url = "http://git.savannah.gnu.org/r/gnulib.git";
     rev = "92b60e61666f008385d9b7f7443da17c7a44d1b1";
-    sha256 = "0xpxq8vqdl0niib961dnsrgjq6kbpyap6nnydzp15dvzfhzgz189";
+#    sha256 = "0xpxq8vqdl0niib961dnsrgjq6kbpyap6nnydzp15dvzfhzgz189";
+     sha256 = "0sa1dndvaxhw0zyc19al5cmpgzlwnnznjz89lw1b4vj3xn7avjnr";
   };
 
   installPhase = "mkdir -p $out; mv * $out/";
